sig-and-wg/sig-core/decisions/dr-020-Kyma-installer-with-embedded-charts.md

```diff
@@ -0,0 +1,16 @@
+# DR 020: Kyma Installer with embedded Kyma charts
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Context
+
+[Proposal](../proposals/kyma-installer-with-charts.md)
```

Comment:
`Find the context of this decision record in [this](../proposals/kyma-installer-with-charts.md) proposal.`

Reply:
Done